mdcdev.js.org

galleniz opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```mdcdev.js.org``` to make it work.
I'll merge after we have "contact". Please keep this PR open.

galleniz commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```mdcdev.js.org``` to make it work.
> 
> I'll merge after we have "contact". Please keep this PR open.
> 
> FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages

I just did it, I added the "mdcdev.js.org" as a subdomain :D yay

![image](https://github.com/js-org/js.org/assets/76523353/bdc2994e-06b3-4008-b146-64209ed49efe)

indus commented:
Welcome https://mdcdev.js.org
